pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.597</version><!-- which version of Jenkins is this plugin built against? Users must have at least this Jenkins version to use this plugin. -->
+    <version>1.609</version><!-- which version of Jenkins is this plugin built against? Users must have at least this Jenkins version to use this plugin. -->
```

Comment:
I'm 99% sure it isn't required.  My development workflow (which maybe isn't the best one?) is to `mvn hpi:run` and test from there.  Bumping to to the latest LTS was easier than figuring out how to install old versions of the git plugin and friends.